src/api/BuildkiteClient.ts

```diff
@@ -255,8 +254,7 @@ export class BuildkiteClient implements BuildkiteAPI {
 
   private getCommonHeaders(contentType?: boolean): Record<string, string> {
     const headers: Record<string, string> = {
-      'X-Buildkite-Source': 'backstage-plugin',
-      'X-Buildkite-Plugin-Version': VERSION,
+      'User-Agent': 'buildkite backstage plugin',
```

Comment:
```suggestion
      'User-Agent': `buildkite-backstage/${VERSION}`,
```
That should then set it to `buildkite-backstage/0.6.2` etc